### ISSUE TITLE
As this project is dead, there's an alternative

### ISSUE THREAD

**Issue by rphlrmblt** [canton7/SyncTrayzor#795]
There's not commit here for more 2 years, so if you want a much better support and a program using a syncthing lib up to date, you can try : https://github.com/Martchus/syncthingtray

**Comment by Jawabiscuit** [comment]
Chrome and Windows virus protection detect malicious code.

**Comment by Tblue** [comment]
Indeed, on VirusTotal, [36/62 engines detect a threat](https://www.virustotal.com/gui/file/5baefe1749c7f7e1ec7c81c83380eb9719a44ea988de1d17485c276eee7ac883).

**Comment by drannex42** [comment]
Not sure why the above are seeing threats, there are no threats in the repository.

**Comment by ProactiveServices** [comment]
> Chrome and Windows virus protection detect malicious code.

Please continue to report false positives to any vendors flagging legit software.

**Comment by rphlrmblt** [comment]
Yep, there was some false positives in prior versions, but with the latest version (1.4.8 at time of writing these comment), it seems OK with Windows Defender.

**Comment by Skaronator** [comment]
syncthingtray 1.4.9 is still blocked for me by chrome and windows defender:
![image](https://github.com/canton7/SyncTrayzor/assets/2886913/d18c8020-f0d3-4435-9a49-376f59ef7275)

**Comment by Tblue** [comment]
I think it makes more sense to continue this discussion over in Martchus/syncthingtray#207.

**Comment by Martchus** [comment]
I'm the author of Syncthing Tray. Thanks for linking it.

It would indeed make sense to continue the discussion there. Unfortunately there's not that much I can do. This is a false-alert so the AV software is at fault and not Syncthing Tray.

**Comment by rphlrmblt** [comment]
> I think it makes more sense to continue this discussion over in [Martchus/syncthingtray#207](https://github.com/Martchus/syncthingtray/issues/207).

Agree for that, but for people who want to have try, the two latest (1.10 and 1.4.11) versions can be downloaded without triggering a false thread from Windows Defender.

**Comment by Martchus** [comment]
Note that the linked Syncthing Tray runs natively on Windows and other platforms it supports. Of course it uses abstractions (Qt) but so did SyncTrayzor (C# runtime / .NET / WPF). Both only complement the official web UI (which one normally wouldn't be running constantly).

**Comment by friedrica137** [comment]
你好，我不清楚如何从synctrayzor迁移到syncthingtray，因为我到现在还在用synctrayzor。

在 2024-07-12 15:54:36，"Eagle Xiang" ***@***.***> 写道：


怎么从 SyncTrayzor 迁移到 SyncthingTray？我希望可以保持过去的目录共享配置
